src/rgw/rgw_lc.cc

```diff
@@ -195,8 +195,10 @@ bool RGWLC::if_already_run_today(time_t& start_date)
   localtime_r(&start_date, &bdt);
 
   if (cct->_conf->rgw_lc_debug_interval > 0) {
-	  /* We're debugging, so say we can run */
-	  return false;
+    if (now - start_date < cct->_conf->rgw_lc_debug_interval)
```

Comment:
I think debug is more proper.  If you set ExipationDays = 1, it means those objects will expire a day later. By setting rgw_lc_debug_interval, the objects will expire 1*rgw_lc_debug_interval. It just for debug.

Here just to make sure in a debug_interval bucket_lc_prepare only do once.